lib/ts/recipe/multifactorauth/index.ts

```diff
@@ -0,0 +1,148 @@
+/* Copyright (c) 2021, VRAI Labs and/or its affiliates. All rights reserved.
+ *
+ * This software is licensed under the Apache License, Version 2.0 (the
+ * "License") as published by the Apache Software Foundation.
+ *
+ * You may not use this file except in compliance with the License. You may
+ * obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+import Recipe from "./recipe";
+import { RecipeInterface, APIOptions, APIInterface } from "./types";
+import { MultiFactorAuthClaim } from "./multiFactorAuthClaim";
+import { SessionContainerInterface } from "../session/types";
+import Multitenancy from "../multitenancy";
+import { getUser } from "../..";
+import UserMetadataRecipe from "../usermetadata/recipe";
+import { getUserContext } from "../../utils";
+
+export default class Wrapper {
+    static init = Recipe.init;
+
+    static MultiFactorAuthClaim = MultiFactorAuthClaim;
+
+    static async getFactorsSetupForUser(tenantId: string, userId: string, userContext?: Record<string, any>) {
```

Comment:
add a comment explaining why we have tenantId here as an input even though it's not used in this function at all